CountItems/CountManager.js

```diff
@@ -61,6 +66,27 @@ class CountManager {
                 }
             });
         }
+        if (results.dataMetrics
+            && results.dataMetrics.bucket
+            && results.dataMetrics.location
+            && results.dataMetrics.account) {
+            Object.keys(results.dataMetrics).forEach(metricLevel => {
+                Object.keys(results.dataMetrics[metricLevel]).forEach(resourceName => {
+                    if (this.store.dataMetrics[metricLevel][resourceName]) {
+                        Object.keys(this.store.dataMetrics[metricLevel][resourceName]).forEach(subMetric1 => {
+                            Object.keys(this.store.dataMetrics[metricLevel][resourceName][subMetric1]).forEach(subMetric2 => {
+                                this.store.dataMetrics[metricLevel][resourceName][subMetric1][subMetric2]
+                                    += results.dataMetrics[metricLevel][resourceName][subMetric1][subMetric2];
+                            });
+                        });
+                    } else {
+                        this.store.dataMetrics[metricLevel][resourceName] = results.dataMetrics[metricLevel][resourceName];
+                    }
+                });
+            });
+        } else {
+            this.store.dataMetrics = results.dataMetrics;
+        }
```

Comment:
It would be great to have a comment here, to explain this logic and why it is is needed.

Reply:
This commend might apply here as well https://github.com/scality/Arsenal/pull/2055/files#r1054615483